### DISCUSSION TITLE
is there a way i can tell picom to always use the desktop background for my terminal

### DISCUSSION THREAD

**Discussion by ymolists** [yshui/picom#1291] [Q&A] [answered]
Hi folks.

I really like picom. thank you for making this tool.

i am using picom with i3 on nix. When i stitch to my terminal i want to use the desktop backgroud. Right now if i have both vscode and tilix open in a workspace in a tabbed fashion the background of tilix is a blurred version of what was displayed on vscode. is there a way i can force all apps to use the desktop background instead ?

Thank you !!!

**Comment by absolutelynothelix** [comment, marked as answer]
`transparent-clipping`.
